chapters/data-processing.tex

```diff
@@ -0,0 +1,784 @@
+%------------------------------------------------
+
+\begin{fullwidth}
+
+% Chapter overview
+The tasks described in this chapter are called by many different names.
+Some would say data cleaning, others data munging, or maybe data wrangling.
+It means transforming raw data into a more convenient format for your intended use.
+Although original data typically requires more extensive data cleaning than secondary data,
+it is broadly agreed that this is, on average, the most time-consuming step in a project's data work \citep{dasu2003exploratory}.
```

Comment:
Suggested rephrase: 'This is the most time-consuming step of a project's data work, particularly for projects using raw data.\citep{dasu2003exploratory}.

Reply:
Can this be said and cited after the full-width section? It feels like a typo to all of a sudden have an in-line citation.